### PR TITLE
Add argument parser for Classfication type

### LINKED ISSUE #143: Add an argument for train.py to choose between training for Y vs D+N or Y+D vs N

### PR DESCRIPTION
**Pull request by Shashi456**
Fixes #143.

### CODE REVIEW

**Comment by Shashi456** [comment]
@marco-c , can you take a look at this?

File: autowebcompat/network.py [hunk 1]
```diff
@@ -10,6 +10,7 @@
     'nadam': Nadam(),
     'rms': RMSprop()
 }
+CLASSIFICATION_TYPE = ['Y vs N + D', 'Y + D vs N']
```
Comment (marco-c) [inline autowebcompat/network.py]: `CLASSIFICATION_TYPE` doesn't belong in this module, as this module is only about neural network-related stuff.

**Comment by marco-c** [comment]
Please rebase on top of master, the commit history is getting a little to large and it is starting to hinder reviewability.

File: autowebcompat/network.py (hunk 1 above)
Comment (Shashi456) [reply autowebcompat/network.py]: @marco-c do you have any name in mind for this

Comment (marco-c) [reply autowebcompat/network.py]: The name is OK (maybe use the plural). It's the module where you put it that isn't OK.

File: train.py [hunk 2]
```diff
@@ -3,9 +3,13 @@
 
 from autowebcompat import network, utils
 
+CLASSIFICATION_TYPES = ['Y vs N + D', 'Y + D vs N']
```
Comment (marco-c) [inline train.py]: I'd move this to utils.py, right before the `to_categorical_label` function.

File: train.py [hunk 3]
```diff
@@ -3,9 +3,13 @@
...
+
+
 parser = argparse.ArgumentParser()
 parser.add_argument('network', type=str, choices=network.SUPPORTED_NETWORKS, help='Select the network to use for training')
 parser.add_argument('optimizer', type=str, choices=network.SUPPORTED_OPTIMIZERS, help='Select the optimizer to use for training')
+parser.add_argument('classification_type', type=str, choices=CLASSIFICATION_TYPES, help='Select the classification_type for training')
```
Comment (marco-c) [inline train.py]: Let's add a default here, using `CLASSIFICATION_TYPES[0]`.

File: train.py [hunk 4]
```diff
@@ -3,9 +3,11 @@
 
 from autowebcompat import network, utils
 
+
```
Comment (marco-c) [inline train.py]: Nit: no newline here

**Pull request merged**